templates/common/on-prem/units/on-prem-dnsmasq.yaml

```diff
@@ -0,0 +1,10 @@
+name: on-prem-dnsmasq.service
```

Comment:
I'm not sure we actually need a separate service for this now that it's being run as vanilla dnsmasq. I'm also not sure if there's a simple way to enable an existing OS service using MCO. I'm open to suggestions.

Reply:
I would prefet to have a separate service or a systemctl override of the default one.